assets/js/modules/reader-revenue-manager/components/setup/SetupMain.stories.js

```diff
@@ -27,8 +32,27 @@ function Template() {
 
 export const Default = Template.bind( {} );
 Default.storyName = 'Default';
+Default.scenario = {
+	label: 'Modules/ReaderRevenueManager/Setup/SetupMain/Default',
```

Comment:
We should avoid hardcoding the `label` as this is constructed based on other values such as the `storyName` and the `default title` below.

Reply:
Thanks - another interesting comment. Most of our stories do specify the label, but you are quite right, it shouldn't be needed.

In fact I've only found one scenario without a label in the codebase (in fact it's one that you updated recently to remove the label :)).

https://github.com/google/site-kit-wp/blob/5e0d5395472431bdc7a9796ea6db1e56bbb668b4/assets/js/modules/ads/components/settings/SettingsView.stories.js#L49

Anyhow - another good shout, I've removed the label. Again, it would be nice to tidy the existing stories up at some point in this regard.

Reply:
Yeah, I did some analysis recently and found that there are a lot of stories where the generated scenario label differed from the one explicitly referenced but yeah it would be nice to clean that up.